packages/sdk/src/auth/signers/aptos-tx-token-signer.ts

```diff
@@ -0,0 +1,20 @@
+import type { TokenSigner, TokenSignerResult } from "@auth/auth.interface";
+import type { DialectWalletAdapterWrapper } from "@dialectlabs/sdk/lib/types/wallet-adapter/dialect-wallet-adapter-wrapper";
+import type { WalletAddress } from "@wallet/internal/wallet-address";
+
+export abstract class AptosTxTokenSigner implements TokenSigner {
```

Comment:
I think we need to double-check, but may be it's easier to support token signature via https://github.com/aptos-labs/aptos-core/blob/bbe8eed457c17c90456824148cdd8abca3751fdd/ecosystem/typescript/sdk/src/aptos_account.ts#L90 instead of transaction